moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h

```diff
@@ -301,7 +301,7 @@ class TrajectoryExecutionManager
   void loadControllerParams();
 
   // Name of this class for logging
-  const std::string name_ = "trajectory_execution_manager";
+  static const std::string LOGNAME = "trajectory_execution_manager";
```

Comment:
Getting closer :laughing: Other classes define this as a non-member in the cpp file

Reply:
Okay